website/public/locales/en/leaderboard.json

```diff
@@ -24,6 +24,8 @@
   "reply": "Replies",
   "reply_ranked_1": "Replies ranked first",
   "score": "Score",
+  "show_active_user": "Show active users",
+  "show_banned_user": "Show banned users",
```

Comment:
are those only visible for admins  & moderators? if yes, we don't need to bother the community with translating them, we can have them in the page directly.

Reply:
Yes, the trollboard is only visible for mods and admins, but since the trollboard table columns are translated, I added this radio too

Reply:
@ArgiesDario Can you revert these translations and inline them? Translation for admin probably will be removed soon.

Reply:
Removed